### ISSUE TITLE
when rezzing an object it floats

### ISSUE THREAD

**Issue by TabbyCat2130** [cinderblocks/radegast#168]
My sister informed me that anytime I rezz objects they float in the air and don't tuch the ground. Using a screen reader their is no way to bring them to the ground. She has to move them down for me. Resaults should be when rezzing an object that it rezzs on the ground and not in the air.

**Comment by cinderblocks** [comment]
The viewer itself doesn't know where the ground is.
